# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/dhcp6c/hotfix.patch
# Copyright (C) 2019 - 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- dhcp6c-20200512/dhcp6c.c.vanilla	2023-03-03 17:00:10.321621785 +0100
+++ dhcp6c-20200512/dhcp6c.c	2023-03-03 17:03:17.317620815 +0100
@@ -27,6 +27,10 @@
  * OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
  * SUCH DAMAGE.
  */
+
+#define __UAPI_DEF_IN6_PKTINFO 1
+#include <linux/ipv6.h>
+
 #include <sys/types.h>
 #include <sys/param.h>
 #include <sys/socket.h>
